src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1128,8 +1128,25 @@
             (munge-uuid "transaction_uuid") ", environment_id, status_id
             FROM reports"))
 
+    (sql/do-commands
```

Comment:
it's a pretty heavy migration that recreates the same tables. If we can stomach splitting it all out I've got it in that form somewhere -- I think having them separate is far preferable in general.